regexes/client/mobile_apps.yml

```diff
@@ -185,7 +185,7 @@
 - regex: 'BingWeb/([\d\.]+)?'
   name: 'BingWebApp'
   version: '$1'
-- regex: 'GSA/([\d\.]+)?'
+- regex: '(?:com.google.GoogleMobile|GSA)/([\d\.]+)?'
```

Comment:
note:
the maximum version I could find in the logs is dated 2019.02
`google. com. Google Mobile/67.1.0 for iPhone/12.1.3 for iPhone7_2`

if compare version 67 with the release date 2019.02.16 and take useragent from the log 2019.02 -  then everything fits together

Website https://iphone.apkpure.com/google/com.google.googlemobile there I looked at the app versions ( appstore is limited to the list of versions)